superbench/benchmarks/micro_benchmarks/cudnn_test/cudnn_helper.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @file cudnn_helper.cpp
+ * @brief  Cpp file for some functions related to cudnn
+ */
+
+#include <bits/stdc++.h>
+#include <stdlib.h>
```

Comment:
are these two libs used in this file?

Reply:
modified to <numeric> and <stdlib.h>, I need to use them for std::rand() and std::accumulate()

Reply:
renamed to <cstdlib>